cmd/krel/cmd/release_notes.go

```diff
@@ -113,12 +141,85 @@ func runReleaseNotes() (err error) {
 	logrus.Infof("Using start tag %v", start)
 	logrus.Infof("Using end tag %v", tag)
 
-	_, err = releaseNotesFrom(start)
+	if releaseNotesOpts.sigReleaseFork == "" && releaseNotesOpts.jsonFile == "" && releaseNotesOpts.mdFile == "" {
```

Comment:
This should probably be a small function, like: `validatePROptions()`
Add a `BoolVar` flag like `--create-pr` and then you can do something like:
```go
if releaseNoteOpts.createPR {
  prOptsErr := validatePROptions(releaseNotesOpts)
  if prOptsErr != nil {
    return prOptsErr
  }
}
```

Reply:
Done. The new flag is called `--create-draft-pr` as we could eventually have another for the JSON version of the release notes.

Reply:
Done, `validatePROptions` now exists